Caculator/app/src/main/res/layout/activity_main.xml

```diff
@@ -4,9 +4,293 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:weightSum="20"
+    android:background="@color/white"
     android:orientation="vertical"
-    tools:context=".MainActivity"
-    android:background="#D6D5D5">
+    android:weightSum="20"
+    tools:context=".MainActivity">
+
+    <EditText
+        android:id="@+id/editText"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="7"
+        android:background="@color/white"
+        android:editable="false"
+        android:gravity="end|bottom"
+        style="@style/MyButton"
+        android:textSize="35sp" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="7"
+        android:background="@color/white"
+        android:gravity="end|bottom"
+        android:text="0"
+        style="@style/MyButton" />
+
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_weight="1.2"
+        android:orientation="horizontal"
+        android:weightSum="10"
+        android:background="@color/background">
+
+        <Button
+            android:id="@+id/btnDelete"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_margin="0.5dp"
+            android:layout_weight="2.5"
+            android:background="@drawable/custom_button"
+            android:text="C"
+            android:textColor="#ff00"
```

Comment:
màu cũng cho vào trong colors.xml nhé Việt 
Nhớ đặt theo tên của màu 
http://chir.ag/projects/name-that-color/#FFFF00
```suggestion
            android:textColor="@color/color_yellow"
```